src/view/writer.js

```diff
@@ -72,6 +74,57 @@ export function breakRange( range ) {
 }
 
 /**
+ * Breaks {@link engine.view.ContainerElement container view element} into two, at the given position. Position
+ * has to be directly inside container element and cannot be in root. Does not break if position is at the beginning
+ * or at the end of it's parent element.
+ *
+ *		<p>foo^bar</p> -> <p>foo</p><p>bar</p>
+ *		<div><p>foo</p>^<p>bar</p></div> -> <div><p>foo</p></div><div><p>bar</p></div>
+ *		<p>^foobar</p> -> ^<p>foobar</p>
+ *		<p>foobar^</p> -> <p>foobar</p>^
+ *
```

Comment:
Also, we may consider renaming `breakAt` to `breakAttributes` now to stress the difference.